src/compas_rhino/install.py

```diff
@@ -106,11 +115,40 @@ def install(version=None, packages=None):
         if status != 'OK':
             exit_code = -1
 
+    if len(installed_packages):
+        print()
+        print('Running post-installation steps...')
+        print()
+        _run_post_execution_steps(after_rhino_install(installed_packages))
+
     print('\nCompleted.')
     if exit_code != 0:
         sys.exit(exit_code)
 
 
+def _run_post_execution_steps(steps_generator):
+    post_execution_errors = []
+    for result in steps_generator:
+        if isinstance(result, Exception):
+            post_execution_errors.append(result)
+            continue
+
+        for item in result:
+            try:
+                package, message, success = item
+                status = 'OK' if success else 'ERROR'
```

Comment:
should an error here result in an `exit_code = -1`?

Reply:
you're right, fixed at https://github.com/compas-dev/compas/pull/770/commits/7b6ef0dba37bc879b4c26c0781dd2e6e8cfee326

Reply:
I think it's a little more complicated.  Shouldn't all the successes be collected into some list `successes` and then instead of returning True, `all(successes)` is returned?